cmd/grr/workflow.go

```diff
@@ -24,16 +25,14 @@ func getCmd(registry grizzly.Registry) *cli.Command {
 
 func listCmd(registry grizzly.Registry) *cli.Command {
 	cmd := &cli.Command{
-		Use:   "list <jsonnet-file>",
+		Use:   "list [-d <directory> [<jsonnet-file>]",
 		Short: "list resource keys from file",
 		Args:  cli.ArgsExact(1),
 	}
-	opts := grizzlyOptsFromCmd(cmd)
+	opts := grizzly.GrizzlyOpts{}
```

Comment:
```suggestion
	var opts grizzly.GrizzlyOpts
```

This form is very common for "just initializing". Also, why isn't that `grizzly.Opts` which reads nicer?

Reply:
sure, adjusted.